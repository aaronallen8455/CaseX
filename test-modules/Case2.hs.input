module Case1 where

main :: IO ()
main = pure ()

test :: Maybe Bool -> Bool
test x = case x of
  -- 1
  Nothing -> False
  -- 2
  Just SPLIT -> True
  -- 3
