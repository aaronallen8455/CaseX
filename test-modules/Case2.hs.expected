module Case1 where

main :: IO ()
main = pure ()

test :: Maybe Bool -> Bool
test x = case x of
  -- 1
  Nothing -> False
  -- 2
  Just False -> True
  Just True -> True
  -- 3
